src/com/google/javascript/jscomp/AbstractCommandLineRunner.java

```diff
@@ -2013,6 +2017,19 @@ private void outputBundle() throws IOException {
     outputManifestOrBundle(config.outputBundles, false);
   }
 
+  @GwtIncompatible("Unnecessary")
+  private void outputInstrumentationParamMapping() throws IOException {
+
+    if (!Strings.isNullOrEmpty(config.instrumentationParamMappingFile)) {
+      if (compiler.getOptions().getInstrumentForCoverageOption() != InstrumentOption.PRODUCTION) {
+        throw new FlagUsageException("Expected --instrument_code to be passed with PRODUCTION " +
```

Comment:
That check is more appropriate in next to the place where we parse flags.

Reply:
Moved.